src/main/java/org/broadinstitute/hellbender/tools/walkers/validation/Concordance.java

```diff
@@ -291,8 +290,12 @@ public Object onTraversalSuccess() {
     @Override
     protected boolean areVariantsAtSameLocusConcordant(final VariantContext truth, final VariantContext eval) {
         final boolean sameRefAllele = truth.getReference().equals(eval.getReference());
+
         // we assume that the truth has a single alternate allele
-        final boolean containsAltAllele = eval.getAlternateAlleles().contains(truth.getAlternateAllele(0));
+        final boolean containsAltAllele =
```

Comment:
You need to update the comment on line 294.

Reply:
Fixed!